lib/index.js

```diff
@@ -35,7 +37,7 @@ function getVODMeta(vodID) {
 }
 
 function getChatFragment(start, vodID) {
-    return fetch("https://rechat.twitch.tv/rechat-messages?start="+start+"&video_id="+vodID)
+    return _fetch("https://rechat.twitch.tv/rechat-messages?start="+start+"&video_id="+vodID)
```

Comment:
I don't think the Client ID should be passed to rechat.twitch.tv. IT shouldn't even be used, so it's better to try and match Twitch's usage (even though it still clearly is different due to missing referrer etc.)